.github/workflows/ci.yaml

```diff
@@ -140,6 +140,10 @@ jobs:
           gotestsum_version: 0.4.1
       - name: Check kind cluster
         run: |
+          docker build cmd/nse -t networkservicemeshci/nse:master
+          docker build cmd/fake-nsmgr -t networkservicemeshci/fake-nsmgr:master
```

Comment:
Why are we using a fake-nsmgr instead of the real one?

Reply:
@edwarnicke we'll replace this fake-nsmgr with real. As I know @haiodo already adding real nsmgr tests.